src/docs/truffle/advanced/build-processes.md

```diff
@@ -4,6 +4,10 @@ layout: docs.hbs
 ---
 # Build Processes
 
+<p class="alert alert-warning">
+**Alert**: The `build` command and this approach is being deprecated. Please use third-party build tools like webpack or grunt, or see our [Truffle Boxes](/boxes) for an example.
```

Comment:
@gnidan Are we actually deprecating this?

Reply:
@eggplantzzz I think it's reasonable to say it's deprecated, since it was originally added as a front-end build tool, but what do you think?